server/pkg/integrationapi/project.go

```diff
@@ -2,17 +2,69 @@ package integrationapi
 
 import (
 	"github.com/reearth/reearth-cms/server/pkg/project"
+	"github.com/reearth/reearthx/account/accountdomain/workspace"
 	"github.com/samber/lo"
 )
 
 func NewProject(p *project.Project) Project {
+	var publication *ProjectPublication = nil
+	if p.Publication() != nil {
+		publication = &ProjectPublication{
+			Scope:       ToProjectPublicationScope(p.Publication().Scope()),
+			AssetPublic: lo.ToPtr(p.Publication().AssetPublic()),
+			Token:       lo.ToPtr(p.Publication().Token()),
+		}
+	}
+
+	var requestRoles *[]ProjectRequestRole = nil
+	if p.RequestRoles() != nil {
+		r := lo.FilterMap(p.RequestRoles(), func(r workspace.Role, _ int) (ProjectRequestRole, bool) {
+			role := ToRequestRole(r)
+			if role != nil {
+				return *role, true
+			}
+			return "", false
+		})
+		requestRoles = &r
+	}
+
 	return Project{
-		Id:          p.ID().Ref(),
-		WorkspaceId: p.Workspace().Ref(),
-		Name:        lo.ToPtr(p.Name()),
-		Description: lo.ToPtr(p.Description()),
-		Alias:       lo.ToPtr(p.Alias()),
-		CreatedAt:   lo.ToPtr(p.CreatedAt()),
-		UpdatedAt:   lo.ToPtr(p.UpdatedAt()),
+		Id:           p.ID().Ref(),
+		WorkspaceId:  p.Workspace().Ref(),
+		Name:         lo.ToPtr(p.Name()),
+		Description:  lo.ToPtr(p.Description()),
+		Alias:        lo.ToPtr(p.Alias()),
+		Publication:  publication,
+		RequestRoles: requestRoles,
+		CreatedAt:    lo.ToPtr(p.CreatedAt()),
+		UpdatedAt:    lo.ToPtr(p.UpdatedAt()),
+	}
```

Comment:
most of the project fields are required (not nullable), so please update the oapi schema to make them required.